### PR TITLE
fixes: Renaming patient categorisation #7811

### LINKED ISSUE #7811: Renaming patient categorisation
Currently the patient categorisation is as:
1) Comfort Care
2) Stable
3) Abnormal
4) Critical

Change this to
1) Comfort Care- keep as is
2) Stable- "Mild"
3) Abnormal- "Moderate"
4) Critical- keep as is.

This change needs to be made in the following places:

- [x] Consultation and edit consultation form (dropdown for category)
- [x] Log update form (dropdown for category)
- [x] Patient dashboard
- [x] Patient listing
- [x] Advanced filter (category filter)

Once done, inform Neil to change the terms on the state dashboard @nihal467

### PR DESCRIPTION
**Pull request by Sulochan-khadka**
## Proposed Changes

- Fixes #7811 

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by Sulochan-khadka** [comment]
@rithviknishad could you review this?

File: src/Common/constants.tsx [hunk 1]
```diff
@@ -375,16 +375,16 @@ export const RESPIRATORY_SUPPORT = [
...
 export const PATIENT_CATEGORIES: {
   id: PatientCategoryID;
   text: PatientCategory;
   twClass: string;
 }[] = [
   { id: "Comfort", text: "Comfort Care", twClass: "patient-comfort" },
-  { id: "Stable", text: "Stable", twClass: "patient-stable" },
-  { id: "Moderate", text: "Abnormal", twClass: "patient-abnormal" },
+  { id: "Mild", text: "Mild", twClass: "patient-mild" },
+  { id: "Moderate", text: "Moderate", twClass: "patient-moderate" },
```
Comment (rithviknishad) [inline src/Common/constants.tsx:387]: You need not change the ID' and tailwind classess. You can just change the labels (text)

File: src/Common/constants.tsx [hunk 2]
```diff
@@ -683,8 +683,8 @@ export const MOTOR_RESPONSE_SCALE = [
   { value: 6, text: "Obeying commands/Normal acrivity" },
   { value: 5, text: "Moves to localized pain" },
   { value: 4, text: "Flexion/Withdrawal from pain" },
-  { value: 3, text: "Abnormal Flexion(decorticate)" },
-  { value: 2, text: "Abnormal Extension(decerebrate)" },
+  { value: 3, text: "Moderate Flexion(decorticate)" },
+  { value: 2, text: "Moderate Extension(decerebrate)" },
```
Comment (rithviknishad) [inline src/Common/constants.tsx:687]: Why was this changed? Totally unrelated to patient category.

File: src/Components/Patient/ManagePatients.tsx [hunk 3]
```diff
@@ -79,8 +79,8 @@ function TabPanel(props: TabPanelProps) {
 
 const PatientCategoryDisplayText: Record<PatientCategory, string> = {
   "Comfort Care": "COMFORT CARE",
-  Stable: "STABLE",
-  Abnormal: "ABNORMAL",
+  Mild: "MILD",
+  Moderate: "MODERATE",
```
Comment (rithviknishad) [inline src/Components/Patient/ManagePatients.tsx:83]: Keys need not be changed. Just display text needs to be changed

File: src/Components/Patient/PatientCategorySelect.tsx [hunk 4]
```diff
@@ -24,7 +24,7 @@ export default function PatientCategorySelect(
               "h-2 w-2 rounded-full",
               {
                 Comfort: "bg-patient-comfort",
-                Stable: "bg-patient-stable",
+                Mild: "bg-patient-stable",
```
Comment (rithviknishad) [inline src/Components/Patient/PatientCategorySelect.tsx:27]: same here too

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
Closing as completed in https://github.com/coronasafe/care_fe/commit/1554ef2fc6af55333199140659468e894b220dda

**Pull request closed without merging**